fineract-provider/src/main/java/org/apache/fineract/cob/loan/LoanCOBManagerConfiguration.java

```diff
@@ -57,8 +57,6 @@ public class LoanCOBManagerConfiguration {
     @Autowired
     private PlatformTransactionManager transactionManager;
     @Autowired
-    private RemotePartitioningManagerStepBuilderFactory stepBuilderFactory;
```

Comment:
Why do you change this?

Reply:
This is not needed more

Reply:
Injecting the factory is better than creating manually. Please undo this change!

Reply:
The factory is deprecated, https://github.com/spring-projects/spring-batch/pull/4582/files

Reply:
RemotePartitioningManagerStepBuilderFactory is not deprecated. Please dont remove it!

Reply:
Change reverted